### ISSUE TITLE
Add option to pass connect_four_neighbors_only option to Circuitscape

### ISSUE THREAD

**Issue by pkenya** [Circuitscape/Omniscape.jl#60]
Hi all, thanks for a great package, enjoying using it.  Is it possible to pass and argument to circuitscape to limit connectivity to connect to four neighbours only, rather than the default 8?

i.e in Circuitscape you can set connect_four_neighbors_only = True

Cheers,

Pete

**Comment by vlandau** [comment]
Glad you're enjoying the package @pkenya! For now this is not possible in Omniscape but I may be able to add that possibility somewhat easily assuming that Circuitscape in Julia carried that feature over from Circuitscape.py. I'll double check that Circuitscape has that feature still, then if so, I'll explore adding it to Omniscape. Will post back here with what I find.

**Comment by vlandau** [comment]
Okay, I've got this working on the [vl/fourneighbors](https://github.com/Circuitscape/Omniscape.jl/tree/vl/fourneighbors) branch. Would you mind testing with this branch to confirm it is working for you as expected? Add `connect_four_neighbors_only = true` to your Omniscape .ini to set the option.

You can install Omniscape from the vl/fourneighbors branch by running to following in Julia (make sure you are using Julia 1.5):
```julia
using Pkg; Pkg.add(PackageSpec(name = "Omniscape", rev = "vl/fourneighbors"))
```

Once we verify everything is working I'll go ahead and merge this into the main branch and issue a new release.

Thanks for the feedback and feature request!

**Comment by pkenya** [comment]
Great - will try it now and give feedback.

Thanks!

**Comment by pkenya** [comment]
Working as expected on a small subset of my data, I tested `connect_four_neighbors_only = true ` and `connect_four_neighbors_only = false`, and both produced results that look as you would expect for the cumulative current maps and the normalized ones dependent on the four or eight neighbors. I'm going to run this on my full dataset now. Thanks again

**Comment by vlandau** [comment]
Excellent, thanks! Sorry for the late response here. I'll go ahead and merge that pull request. I have one other addition to make to Omniscape.jl before issuing a new release, so expect this to be available in Omniscape v0.5.0 within the next couple of weeks! Until then, you can install directly from the "main" branch to be able to use this functionality.

**Comment by vlandau** [comment]
This will be released Omniscape v1.0
